src/scripts/components/stories/story-image/story-image.module.styl

```diff
@@ -0,0 +1,4 @@
+.photo
+  flex: 1
+  width: 100%
+  height: 0
```

Comment:
`height: 0` ? Could you explain this pattern?

Reply:
Actually I just copied css props form image gallery styles. I am not sure why `height` needs to be 0.